renderer/components/Settings/messages.js

```diff
@@ -37,4 +37,10 @@ export default defineMessages({
   lndTargetConfirmations_fast_description: 'Number of blocks to target for "fast" sending speed.',
   fieldgroup_general: 'General',
   fieldgroup_wallet: 'Wallet',
+  rateProvider_label: 'Rate provider',
+  rateProvider_option_coinbase: 'Coinbase',
```

Comment:
Do we actually want the company names to be translatable? I did it this was with the block explorers but now question wether there is really any point. Would these company names ever be written differently in different languages?

Reply:
I think we don't. I've used that to make it consistent with how block explorers are handled mostly

Reply:
I've updated rate provider Select to use names directly from the config (which is probably the right thing to do in the first place because previously those weren't used at all)
Also, update blockexplorer to do the same for consistency